src/test/java/com/example/expense/tracker/integrationTest/CategoryIntegrationTest.java

```diff
@@ -0,0 +1,88 @@
+package com.example.expense.tracker.integrationTest;
+
+import com.example.expense.tracker.ExpenseTrackerApplication;
+import com.example.expense.tracker.model.Category;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = ExpenseTrackerApplication.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class CategoryIntegrationTest {
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Test
+    public void testCreateCategory() {
+        // Create a category request object
+        Category category = new Category();
+        category.setName("Test Category");
+
+        // Send HTTP POST request to create a new category
```

Comment:
remove unwanted comments in all the places.

Reply:
unwanted comments removed